### ISSUE TITLE
Get at least 3 clubs submitting feedback to Pakrat

### ISSUE THREAD

**Issue by hellyeah** [hackedu/hackedu#373]
Meetings this week, per the club schedule calendar by @hellyeah:

Goal:

- [ ] 3:15PM PDT Thursday, San Leandro High School (Jiahao) - accidentally sent wrong link
- [x] 3:30PM PDT Thursday, Lowell High School (Jeremy)
- [ ] 11:30PM PDT Wednesday, Cherry Hill East (Bogdan)

Stretch:

- [ ] 4:00PM PDT Tuesday, USC Hybrid School (Gemma)
- [ ] 10:30PM PDT Tuesday, Glenaeon Rudolf Steiner School (Harrison)
- [ ] 12:30PM PDT Friday, Ravenwood High School (Tejas)

**Comment by hellyeah** [comment]
Maybe suggest: "when jonathan did hackcamp he would always forget to ask for feedback, so he set an alarm for 10 minutes before the end of the meeting to collect feedback."

Jon's idea: set precedent of asking for feedback from them first.

**Comment by jonleung** [comment]
# Meeting: Dave and Jonathan Chat About How To Collect Feedback

_In attendance:_

@hellyeah, @jonleung

--------------------------------------------------------------------------------

TLDR Conclusion: It is a lot easier to have people fill out a written feedback
than a online feedback form.

## Online Feedback Form

1. club leader remembers to give feedback
2. club leaders sends out URL to everyone to fill out the feedback
3. club leader needs to make an announcement to the whole class about filling 
   out the feedback form
4. club member needs to open the link to the feedback form
5. club member needs to submit the feedback form

### Notes about the process of collecting online feedback:

- This is a guess but I would imagine that once club members have the feedback
  form open, almost everyone fills it out. The problem is that many club members
  don't even open up the feedback form to begin with. Why? Because opening
  up the feedback form takes a lot of work especially when they are in the
  middle of doing something else. Instead, of the club member having to do the
  work of obtaining the feedback form. it is a lot easier of the club leader
  "pushes" the feedback form to them by handing them a physical paper feedback
  form. 
    - You can view club members having to open up the feedback form themselves
    as many "clients" having to do work and "pull" for something vs just 1
    club leader "pushing" the feedback form to all the clients. If you have
    "lazy" and "distrated" clients, this is a much better solution.
- While club members are supposedly filling out the feedback form, the club
  leader has *no* visibility as to whether club members are filling out feedback
  forms or not.
- Also it is not clear to a club leader when someone finished filling out a
  feedback form by just looking around the classroom (they have to open up
  Packrat and cross reference who has filled out the feedback form and who 
  hasn't and club leaders don't have time to do this while they are running a 
  club). Hence there is no accountability for club members to actually fill out
  the feedback form
- Club members know that club leaders can't tell whether they are filling out a
  feedback form and cannot easily tell if they have submitted and 
  feel they can "get away" with not filling out feedback.

## Written Feedback Form

1. club leader remembers to give feedback
2. club leader  passes around the feedback form to one person at a
   time and _**individually**_ asks everyone to fill it out. The fact that
   each club member is individually asked is key. (By individually it could be
   asking 1 person at a time or 1 table or group a time (2-4 people)).
3. club leader makes an announcement to the whole class about filling 
   out the feedback and why it matters to club members. This is the **second**
   time that club members are hearing about this because they already had
   the one on one interaction. This acts as a second reminder for club members
   to fill out the feedback increasing the likeliness that it would be done.
4. club member fill out the feedback form
5. club leader **individually** collects the feedback forms

### Notes about collecting written feedback:

- Because a club leader **_individually_** has made a request of one club member
  at a time to fill out the feedback form, the club member will more likely
  honor the request.
- There are two reminders to fill out the feedback form instead of one (the 
  individual request and the group announcement)
- It is very clear to club leaders if club members are filling out the feedback
  forms if the club leader just looks around the room (either club members are
  touching their pens to the piece of paper or not).
  Club members will therefore will not feel like they can get away with not
  filling it out.
- It is very clear if the club member submits a feedback form because they need
  to personally hand it to a club leader. The club member would look really
  bad if they didn't have anything to hand to a club leader and so they will
  fill it out with something. You can even treat the feedback form as an "exit 
  ticket", i.e. you are expected to hand the feedback form to the leader
  standing at the door before you can walk out.
    
### Issues

- Just because people are now "forced" to do feedback forms, you won't
necessarily get quality answers. You might get single word responses 
That's why the feedback form should rely on circling numbers on a scale of
-5 to 5. More details below
- We don't get links to projects that people worked on... Not sure how to fix
  this yet. One idea is for people to paste the links to their project in a
  chat...


## Written Feedback

- [Google Doc](https://docs.google.com/document/d/1Hl9I9QtURDDp3IhvKVoP8EIRk4miK88MzkkXPUCvJUs/edit)
- [PDF](http://f.cl.ly/items/2Y2N2Z19051F1n3u1d0C/FeedbackForms%20(1).pdf)

![image](https://cloud.githubusercontent.com/assets/654941/10674732/6bdad410-78b1-11e5-827a-1d25c4f9f669.png)

## Tasks

Dave will reach out to the the following club leaders:

- [ ] @amysorto (talk to co-lead as well)
- [ ] @Bogidon /@salman
- [ ] @JevinSidhu (talk to co-lead as well)
- [ ] @j1lin0101 (talk to co-lead as well)
- [ ] @xeonjake (talk to co-lead as well)
- [ ] Leila/Erina
- [ ] @tejasmanohar (talk to co-lead as well)

2) Dave talks to **2 leaders** at the same time when doing this:
  a) ask them what they want in the club
  b) show how feedback helps them get what they want
  c) give them the link to the Google Doc and the PDF
  d) **ask them for their feedback**
  e) tell them to print out x number of copies
  f) have them choose a specific club leader to take lead on collecting feedback
  g) talk about why it's helpful to set an alarm to do this (jon can help)
  h) ask if they want reminder in call
  i) set reminder or IFTTT for me to call a club leader

**Comment by hellyeah** [comment]
When does the 1on1 first interaction happen?

**Comment by jonleung** [comment]
@hellyeah, when the club leader initially goes around passing out feedback forms individually and asks people to fill out the feedback as they pass it out.

**Comment by hellyeah** [comment]
going to fail this issue this week.

**Comment by hellyeah** [comment]
Failed this issue. Only 2 clubs submitted feedback: Lowell/Project Cipher
